index.js

```diff
@@ -1,3 +1,47 @@
+var library = [{
+  name: "The Paper",
+  cover: "img/book_1.bmp",
+  description: "In this bone-chilling tale of terror from Shamus"
```

Comment:
let's change description to the next type:
description: { 
    short: 'This text will be used for list',
    full: 'This text will be used for modal'
}
Add id to each book.